### PR TITLE
Masq errors

### PR DESCRIPTION
**Pull request by G-Ray**

### CODE REVIEW

File: src/library/masq.js [hunk 1]
```diff
@@ -66,6 +67,13 @@ const openOrCreateDB = (name) => {
   return createPromisifiedHyperDB(name)
 }
 
+const dispatchMasqError = (errorCode) => {
+  const Event = new CustomEvent('MasqError', {
```
Comment (sujeebant) [inline src/library/masq.js]: should be event with lowercase first letter to not confuse it with the Event class

File: src/library/masq.js [hunk 2]
```diff
@@ -380,6 +388,10 @@ class Masq {
       }
 
       this.hub = signalhub(channel, HUB_URLS)
+      this.hub.on('error', () => {
+        dispatchMasqError(MasqError.REPLICATION_SIGNALLING_ERROR)
```
Comment (sujeebant) [inline src/library/masq.js]: I think we should use SIGNALLING_SERVER_ERROR as this is not the instance used for replication

Comment (G-Ray) [reply src/library/masq.js]: It should be used for replicating the profile DB, so I think it's the same error

**Review by sujeebant — APPROVED** [review]

**Review by leventdem — APPROVED** [review]
ok

**Pull request merged**